classes/cluster/mk20_stacklight_advanced/fuel/config.yml

```diff
@@ -8,7 +8,7 @@ classes:
 - system.salt.minion.pki.authority
 - system.salt.minion.pki.certificate
 - system.sphinx.server.doc.reclass
-- system.keystone.client.single
+- system.keystone.client.stacklight
```

Comment:
I'd be happy not to have single/stacklight, better remove ceilometer service definition from single [default] and include it where it is applied.